app/extensions/safe/ffi/authenticator.ts

```diff
@@ -42,8 +42,8 @@ const _decodeReqPool = Symbol( 'decodeReqPool' );
  * characters or symbols which are not valid for a URL like '=' sign,
  * and making it lower case.
  */
-const genAppUri = str => {
-    const urlSafeBase64 = new Buffer( str )
+const genAppUri = ( str ) => {
+    const urlSafeBase64 = new Buffer.from( str )
```

Comment:
I'm surprised that no warnings were thrown for this syntax. Shouldn't it either be `Buffer.from` or `new Buffer`?

Reply:
Ah yeh, probably should be. I'll update this.